createInvoice/src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+spring.mvc.view.suffix= .html
+
+logging.level.org.springframework.web=INFO
+logging.level.guru.springframework.blogs.controllers=INFO
+logging.level.org.hibernate=ERROR
+logging.file=logs/spring-boot-logging.log
+log4j.logger.org.thymeleaf=DEBUG
+
+spring.thymeleaf.cache=false
+
+
+cloudelements.url=https://api.cloud-elements.co.uk
```

Comment:
You may want to put in placeholders for cloudelements.organization and cloudelements.user as a reminder that they need to be populated.

Reply:
is in the readme